<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisitorGUI</class>
 <widget class="QWidget" name="VisitorGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VisitorGUI</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QSplitter" name="splitterWindow">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="traceWidget">
      <layout class="QVBoxLayout">
       <item>
        <widget class="QSplitter" name="splitterStats">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="Q3ListView" name="graphView">
          <column>
           <property name="text">
            <string>Graph</string>
           </property>
           <property name="clickable">
            <bool>false</bool>
           </property>
           <property name="resizable">
            <bool>true</bool>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Time</string>
           </property>
           <property name="clickable">
            <bool>false</bool>
           </property>
           <property name="resizable">
            <bool>false</bool>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Type</string>
           </property>
           <property name="clickable">
            <bool>false</bool>
           </property>
           <property name="resizable">
            <bool>false</bool>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
           <property name="clickable">
            <bool>false</bool>
           </property>
           <property name="resizable">
            <bool>true</bool>
           </property>
          </column>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListView</class>
   <extends>Q3Frame</extends>
   <header>q3listview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>graphView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
